doc/README.rst

```diff
@@ -12,13 +12,7 @@ Building Documentation
 Prerequisites
 =============
 
-Install `Docker <https://www.docker.com/>`_.
-
-Pull the docker container:
-
-.. code-block:: console
-
-    $ docker pull ghcr.io/xmos/doc_builder:main
+Use the `xmosdoc tool <https://github.com/xmos/xmosdoc>`_ either via docker or install it into a pip environment.
```

Comment:
We need to add some information to explain xmosdoc is for XMOS users only. We can add something like we added at the beginning of the readme in fwk_core:

https://github.com/xmos/fwk_core/blob/develop/doc/README.rst

Reply:
Well reminded. Fixed